Gemfile

```diff
@@ -3,6 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in rainforest-cli.gemspec
 gemspec
 
+# Until API bits is updated
+gem 'rainforest', github: 'epaulet/rainforest-ruby', branch: 'rf-test-rfml-id'
```

Comment:
I am not allowed to merge but I made a PR. https://github.com/rainforestapp/rainforest-ruby/pull/2